src/components/Table/readme.md

```diff
@@ -981,6 +981,145 @@ function TableExample(props) {
+        id: '5678zxcvbn',
+    },
+    {
+        name: 'Tahimi Leon',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '9012qwerty',
+    },
+    {
+        name: 'Alejandro',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',  
+        id: '5678zdfgdf',
+    },
+    {
+        name: 'Carlos',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '3434fgfgdf',
+    },
+    {
+        name: 'Luis',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '9012asdfgh',
+    },
+];
+const Title = styled.h1.attrs(props => props.theme.rainbow)`        
+    font-family: 'Lato Light';
+    font-size: 25px;
+    text-align: left;
+    margin-bottom: 10px;
+    color: ${props => props.palette.text.main};
+    `;  
+
+const Subtitle = styled.h3.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 14px;
+    color: ${props => props.palette.text.title};
+    `;
+
+function TableCustomAction(props) {
+    const [data, setData] = useState(props.data);
+
+    function handleOnDeleteElement(id) {
+            console.log(id)
+            const newData = data.filter(e => e.id!==id);
+            setData(newData)
+        }
+    const customActionStyles ={marginLeft: "auto"};
+    const StyledActionCell = styled.div`
+        display: flex;
+        justify-content: center;
+        align-content: center;
+        align-items: center;
+        margin: auto;
+        box-sizing: border-box;
+    `;
+    const CustomAction = ({row})=>(
+            <StyledActionCell>
+                <ButtonIcon
+                        onClick={()=>handleOnDeleteElement(row.id)}
+                        buttonSize="small"
+                        style={customActionStyles}
+                        icon={<FontAwesomeIcon icon={faTrashAlt} />}
+                        />
+            </StyledActionCell>    
+    );
+    return ( 
+        <div className="rainbow-p-bottom_xx-large"> 
+            <GlobalHeader  src="images/user/user3.jpg">
+                <ButtonGroup className="rainbow-m-right_medium">
+                    <ButtonIcon variant="border-filled" disabled icon={<FontAwesomeIcon icon={faPlus} />} />
+                    <ButtonIcon
+                        variant="border-filled"
+                        disabled
+                        icon={<FontAwesomeIcon icon={faEllipsisV} />}
+                    />
+                </ButtonGroup>
+            </GlobalHeader>
+            <div className="rainbow-flex rainbow-justify_spread rainbow-align_center rainbow-p-around_large">
+                    <div>
```

Comment:
I think you don't need this `div` element here
Remember, we should make the examples as simple and clean as possible so we avoid adding noise to the other developers who come to learn how to use the component.